packages/block-library/src/columns/block.json

```diff
@@ -14,9 +14,13 @@
 		],
 		"html": false,
 		"lightBlockWrapper": true,
-		"__experimentalColor": {
-			"gradients": true,
-			"linkColor": true
+		"__experimentalStyles": {
+			"color": {
+				"background": true,
```

Comment:
In the client, the `background` and `text` keys are not used, only support for `__experimentalStyles.color` is checked. Although we could remove them from here and make the server verification do the same, this will make the support keys to have different shape than the block attributes and theme.json. I lean towards making the client be more strict about these checks in a subsequent PR and actually verify that `background` and `text` are true.

Thoughts?

Reply:
See https://github.com/WordPress/gutenberg/pull/24298#discussion_r464115543 as an example where conflating the two is forcing us to do things that we may not want to do.